YourToDo/templates/auth/registration.html

```diff
@@ -2,10 +2,46 @@
 
 {% block content %}
 	<div class="container">
-		<h1>Register</h1>
-		<form action="/registration/" method="post">{% csrf_token %}
-			{{form}}
-			<input type="submit" value="Register" />
-		</form>
+		<div class="form-registration">
+			<h2>Registration</h2>
+			<form action="/registration/" method="post">{% csrf_token %}
+				<div class="username">
+					<label for="{{ form.username.id_for_label }}">Username:</label>
+					{{ form.username }}
+					<p>Length: 30 characters or fewer. Letters, digits and '
+                    '@/./+/-/_ only.'<p>
+                    {{ form.username.errors }}
+				</div>
+				<div class="first_name">
```

Comment:
Each label is surrounded by a div, which is a good idea, but we should instead attach a class that maybe sets the width of the label attributes to a consistent width for better design. Also, most of these classes appear to do nothing.